FewCore/Element.swift

```diff
@@ -116,7 +118,9 @@ public class Element {
 				compareAndSetAlpha(view, alpha)
 			}
 			
-			realizedSelf.element = self
+			if old.frame.size != frame.size {
```

Comment:
`RealizedElements` don't get re-laid-out when when their size changes. Try running the iOS demo with that code turned on and off and tapping "Increase Header Height".

- When it's turned on, each time the size of the header changes, the label is re-centered as it should be.
- When it's turned off, the label stays where it is, making it off-center as the height grows. The header is never marked as needing layout even though it really does.